src/utils/codemods/availableCodemods.js

```diff
@@ -0,0 +1,17 @@
+const { resolveCodemod } = require('./resolveCodemod')
+
+module.exports.availableCodemods = [
+    // contains groups that apply to mutliple repositories
+    ['multi-purpose', []],
+
+    // these should be moved to the respective repository
+    [
+        '@dhis2/d2-ui',
+        [
+            {
+                name: '31.1.0-change-imports.js',
+                path: resolveCodemod('d2-ui'),
+            },
+        ],
+    ],
+]
```

Comment:
I'd prefer to actually move the d2-ui codemod to the d2-ui repository, there's no actual reason to have it in this repo with the new setup and just drop the entire 'internal' codemods functionality until we actually need it. Ideally we never need it because code changes should have an origin of some kind (most likely a library)